webrtc.html

```diff
@@ -3780,6 +3780,7 @@
     readonly        attribute DOMString?              sdpMid;
     readonly        attribute unsigned short?         sdpMLineIndex;
     readonly        attribute DOMString?              foundation;
+    readonly        attribute byte?                   component;
```

Comment:
right but so far all the other fields map 1:1 into the candidate attribute.
And this is supposed to be a convenience helper so maybe it shouldn't try to be too smart?

I have no strong opinion either way